actions/init-ci/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Init CI
+description: Sets up common environment for CI jobs
+
+inputs:
+  buildx:
+    required: false
+    description: Enable buildx system (needed just to enable some docker build-push-action features)
+  gcp_service_account:
+    required: true
+    description: Login to gcloud and gcr with this service account (needed in order to push images and run gcloud)
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+      if: ${{ inputs.buildx }}
+    - name: Authenticate to Google Cloud
+      if: ${{ inputs.gcp_service_account }}
+      uses: google-github-actions/auth@v0
+      with:
+        credentials_json: ${{ inputs.gcp_service_account }}
+    - name: Set up GCloud SDK
+      if: ${{ inputs.gcp_service_account }}
+      uses: google-github-actions/setup-gcloud@v0
+    - name: Configure docker
+      if: ${{ inputs.gcp_service_account }}
+      shell: bash
+      run: gcloud --quiet auth configure-docker
```

Comment:
Maybe remove this `--quiet` so if there's an error we'll be able to debug without running again?

Reply:
Is that a happy or sad "ok"?